### PR TITLE
[DR-70429] remove timezone enums from appeals schemas

### PR DESCRIPTION
**Pull request by anniebtran**
# New schema
We're removing the enum for timezones in the Decision Reviews schemas (SC, HLR, NOD) and letting Lighthouse take care of validating the proper set of timezones so that we don't have to manage them here and on the Lighthouse side. It makes it much easier for us to maintain and reduces errors that arise from an enum being out of date (i.e. the enums defined here... see ticket below for more details about the issues we're seeing)

- [x] Package.json version bumped

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/70429

## Pull Requests to update the schema in related repositories
_After you've merged your changes to vets-json-schema you'll need to make PR's to vets-website and vets-api. Please link them here._

- https://github.com/department-of-veterans-affairs/vets-api/pull/
- https://github.com/department-of-veterans-affairs/vets-website/pull/27046

### CODE REVIEW

File: src/schemas/NOD-create-request-body_v1/schema.json [hunk 1]
```diff
@@ -865,14 +117,7 @@
...
-                      "allOf": [
-                        {
-                          "description": "Veteran's timezone. Used for adding the current date to the signature according to the veteran's local time. If not specified, the current UTC date will be used for the signature (which may be ahead or behind the local date)."
-                        },
-                        { "$ref": "#/definitions/timezone" }
-                      ]
-                    }
+                    "timezone": { "type": "string" }
                   },
                   "required": ["homeless", "phone", "email"],
```
Comment (Mottie) [inline src/schemas/NOD-create-request-body_v1/schema.json:122]: Do we need to add `"timezone"` to this required field?

File: dist/NOD-SHOW-RESPONSE-200_V1-schema.json [hunk 2]
```diff
@@ -945,14 +197,7 @@
...
-                          "description": "Veteran's timezone. Used for adding the current date to the signature according to the veteran's local time. If not specified, the current UTC date will be used for the signature (which may be ahead or behind the local date)."
-                        },
-                        {
-                          "$ref": "#/definitions/timezone"
-                        }
-                      ]
+                      "type": "string"
                     }
                   },
                   "required": [
```
Comment (Mottie) [inline dist/NOD-SHOW-RESPONSE-200_V1-schema.json:203]: Add "timezone" to required?

File: dist/NOD-CREATE-RESPONSE-200_V1-schema.json [hunk 3]
```diff
@@ -945,14 +197,7 @@
...
-                          "description": "Veteran's timezone. Used for adding the current date to the signature according to the veteran's local time. If not specified, the current UTC date will be used for the signature (which may be ahead or behind the local date)."
-                        },
-                        {
-                          "$ref": "#/definitions/timezone"
-                        }
-                      ]
+                      "type": "string"
                     }
                   },
                   "required": [
```
Comment (Mottie) [inline dist/NOD-CREATE-RESPONSE-200_V1-schema.json:203]: Add "timezone" to required?

File: dist/NOD-CREATE-REQUEST-BODY_V1-schema.json [hunk 4]
```diff
@@ -945,14 +197,7 @@
...
-                          "description": "Veteran's timezone. Used for adding the current date to the signature according to the veteran's local time. If not specified, the current UTC date will be used for the signature (which may be ahead or behind the local date)."
-                        },
-                        {
-                          "$ref": "#/definitions/timezone"
-                        }
-                      ]
+                      "type": "string"
                     }
                   },
                   "required": [
```
Comment (Mottie) [inline dist/NOD-CREATE-REQUEST-BODY_V1-schema.json:203]: Add "timezone" to required?

File: dist/HLR-SHOW-RESPONSE-200_V1-schema.json [hunk 5]
```diff
@@ -196,753 +196,7 @@
                     },
                     "timezone": {
                       "type": "string",
```
Comment (Mottie) [inline dist/HLR-SHOW-RESPONSE-200_V1-schema.json:198]: Add "timezone" to required? Not sure where the required field is...

**Review by Mottie — APPROVED** [review]

**Comment by anniebtran** [comment]
@Mottie I didn't feel confident enough in how the schemas are used to validate to make anything required that wasn't required before 🥴 but if you're sure of where I should be adding required values, I can go ahead and do that. 

I added a note in a few of the files but to summarize what I found — I tried to see if we could implement test specs for these schemas but they're using a structural version that isn't supported by the testing tool we're using for other specs.

**Pull request merged**